### ISSUE TITLE
Giving warning "cannot find implementation or library stub for module" when source code in a sub folder

### ISSUE THREAD

**Issue by qiujiangkun** [leinardi/mypy-pycharm#66]
It warns `cannot find implementation or library stub for module` when I put my source code in a sub folder, while running directly `mypy pacro` will show no warnings.
```log
$ tree
├── mypy.ini
├── pacro
│   ├── ast_nodes.py
│   ├── basic_interpreter.py
│   ├── basic_lexer.py
│   ├── basic_parser.py
│   ├── lexer_config.py
│   ├── lexer_token.py
│   ├── main.py
│   ├── pacro.py
│   ├── token_types.py
│   └── utils.py
└── README.md
$ cat mypy.ini
[mypy]
mypy_path = 'pacro'

$ mypy pacro
Success: no issues found in 10 source files

```

**Comment by hstock** [comment]
I have the same issue. It works when doing a manual scan, but the realtime scanner seems to ignore the Python Sources Root folder settings.
